tools/ExampleExtractor/ExampleAnnotationGrammar.g4

```diff
@@ -0,0 +1,78 @@
+// grammar understood by the ExampleExtractor tool when processing HTML comments containing example annotations.
```

Comment:
I suspect users would find it easier to understand that it's just JSON, and then give the valid properties within the JSON. Let's discuss in the meeting though, as I'm biased.

Reply:
On the 2022-11-2 TG2 call, we agreed to keep the ANTLR notation, and Jon will add more words about JSON in a separate PR.